python/fusion_engine_client/messages/measurements.py

```diff
@@ -509,3 +509,130 @@ def to_numpy(cls, messages):
+        # values will be `NAN`.
+        ##
+        self.position_std_enu_m = np.full((3,), np.nan)
+
+        ##
+        # The heading between the primary device antenna and the secondary (in degrees) with
+        # respect to true north.
+        #
+        # @note
+        # Reported in the range [0, 360).
+        #
+        ##
+        self.heading_true_north_deg = np.nan
+
+        ##
+        # The estmated distance between primary and secondary antennas (in meters)
+        #
+        ##
+        self.baseline_distance_m = np.nan
+
+    def pack(self, buffer: bytes = None, offset: int = 0, return_buffer: bool = True) -> (bytes, int):
+        initial_offset = offset
+        if (buffer is None):
+            buffer = bytearray(self.calcsize())
+        buffer = self.timestamps.pack(buffer)
+        offset += self.timestamps.calcsize()
+        self._STRUCT.pack_into(buffer, offset,
+            self.solution_type,
+            self.flags,
+            self.relative_position_enu_m[0],
+            self.relative_position_enu_m[1],
+            self.relative_position_enu_m[2],
+            self.position_std_enu_m[0],
+            self.position_std_enu_m[1],
+            self.position_std_enu_m[2],
+            self.heading_true_north_deg,
+            self.baseline_distance_m)
+        offset += self._STRUCT.size
+        if return_buffer:
+            return buffer
+        else:
+            return offset - initial_offset
+
+    def unpack(self, buffer: bytes, offset: int = 0) -> int:
+        initial_offset = offset
+
+        offset += self.timestamps.unpack(buffer, offset)
+        (solution_type_int,
+            self.flags,
+            self.relative_position_enu_m[0],
+            self.relative_position_enu_m[1],
+            self.relative_position_enu_m[2],
+            self.position_std_enu_m[0],
+            self.position_std_enu_m[1],
+            self.position_std_enu_m[2],
+            self.heading_true_north_deg,
+            self.baseline_distance_m) = self._STRUCT.unpack_from(buffer, offset)
+        offset += self._STRUCT.size
+        self.solution_type = SolutionType(solution_type_int)
+        return offset - initial_offset
+
+    def __str__(self):
+        return f"""\
+HeadingMeasurement @ {str(self.timestamps.p1_time)}
+  Solution Type: {str(self.solution_type)}
+  Relative position (ENU) (m): {self.relative_position_enu_m[0]:.2f}, {self.relative_position_enu_m[1]:.2f}, {self.relative_position_enu_m[2]:.2f}
+  Position std (ENU) (m): {self.position_std_enu_m[0]:.2f}, {self.position_std_enu_m[1]:.2f}, {self.position_std_enu_m[2]:.2f}
+  Heading (deg): {self.heading_true_north_deg:.2f}
+  Baseline distance (m): {self.baseline_distance_m:.2f}"""
+
+    @classmethod
+    def calcsize(cls) -> int:
+        return cls._STRUCT.size + MeasurementTimestamps.calcsize()
+
+    @classmethod
+    def to_numpy(cls, messages: Sequence['HeadingMeasurement']):
```

Comment:
You're missing an import now that you moved this out of `solution.py`, and it's causing your CI to fail. You'll want to add:
```
from typing import Sequence
```
to the top of the file after `import struct`